sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/samples/SampleMappingClaimsADFS.md

```diff
@@ -0,0 +1,119 @@
+## Sample of Mapping claims using ADFS
+
+Below is sample code for adding a string and string array as two new claims to the response with the NuGet
+```csharp
+
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text.Json;
+
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents;
+using Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.TokenIssuanceStart;
+using Microsoft.Extensions.Logging;
+
+namespace AuthEventsTrigger
+{
+    public static class AuthEventsTrigger
```

Comment:
We run samples as tests and use code snippets in the `.md` files. You can generate code snippets by running the [Update-Snippets.ps1 entra](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#updating-sample-snippets) script. For reference, you can check out a sample [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/appconfiguration/Azure.Data.AppConfiguration/tests/samples/Sample1_HelloWorld.cs#L13) and the corresponding generated code snippet in the `.md` file [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample1_HelloWorld.md#create-a-configurationclient).